book/src/bail-and-ensure.md

```diff
@@ -0,0 +1,18 @@
+# bail! and ensure!
```

Comment:
These docs look good, the only change I'd ask for is to put `bail!` and `ensure!` (whenever they appear with the `!` in backtick code style, as I've done in this comment).